src/iframe/IframeArticlePage.js

```diff
@@ -21,8 +21,26 @@ import { getAllDimensions } from '../util/trackingUtil';
 import PostResizeMessage from './PostResizeMessage';
 import FixDialogPosition from './FixDialogPosition';
 import { SocialMediaMetadata } from '../components/SocialMediaMetadata';
+import { fetchResource } from '../containers/Resources/resourceApi';
+import config from '../config';
 
 class IframeArticlePage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      path: undefined,
+    };
+  }
+
+  componentDidMount() {
+    const resourceId = this.props.location.pathname.split('/')[3];
+    fetchResource(resourceId).then(resource => {
+      this.setState({
+        path: resource.path,
```

Comment:
Det er nok datagrunnlaget som er dårlig i test men det kan jo skje i prod også. Mange ressurser har null i path men har innhold i paths. Så vi bør nok legge inn sjekk der også. Feks:
```suggestion
        path: resource.path || resource.paths?[0],
```